Source/SwiftLintFramework/Helpers/XCTestHelpers.swift

```diff
@@ -0,0 +1,24 @@
+import SourceKittenFramework
+
+private let testFunctionNames: Set = [
+    "setUp()",
```

Comment:
Any reason not to include all the overridable members defined here? https://developer.apple.com/documentation/xctest/xctestcase & https://developer.apple.com/documentation/xctest/xctest